src/main/java/be/seeseemelk/mockbukkit/configuration/GlobalServerConfiguration.java

```diff
@@ -0,0 +1,210 @@
+package be.seeseemelk.mockbukkit.configuration;
+
+import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.NotNull;
+
+public class GlobalServerConfiguration
+{
+
+	private int viewDistance = 10;
+	private LevelType levelType = LevelType.DEFAULT;
+	private boolean generateStructures = true;
+	private boolean allowEnd = true;
+	private boolean allowNether = true;
+	private String updateFolder = "update";
+	private boolean sendChatPreviews = false;
+	private boolean enforceSecureProfiles = true;
+	private boolean onlineMode = true;
+	private boolean allowFlight = false;
+	private boolean hardcore = false;
+	private int maxChainedNeighbourUpdates = 1000000;
+	private Component shutdownMessage = Component.text("Server closed");
+	private int maxWorldSize = 29999984;
+	private int simulationDistance = 10;
+	private boolean hideOnlinePlayers = false;
```

Comment:
Would it be easier to make these enums, and make the getters/setters accept them?

Reply:
Discussed on Discord, but decided against this.